hack/make-rules/test-integration.sh

```diff
@@ -66,16 +66,12 @@ runTests() {
   kube::etcd::start
   kube::log::status "Running integration test cases"
 
-  # export KUBE_RACE
-  #
-  # Enable the Go race detector.
-  export KUBE_RACE="-race"
   make -C "${KUBE_ROOT}" test \
       WHAT="${WHAT:-$(kube::test::find_integration_test_dirs | paste -sd' ' -)}" \
       GOFLAGS="${GOFLAGS:-}" \
       KUBE_TEST_ARGS="--alsologtostderr=true ${KUBE_TEST_ARGS:-} ${SHORT:--short=true} --vmodule=${KUBE_TEST_VMODULE}" \
-      KUBE_RACE="" \
-      KUBE_TIMEOUT="${KUBE_TIMEOUT}"
+      KUBE_TIMEOUT="${KUBE_TIMEOUT}" \
+      KUBE_RACE=""
```

Comment:
does this order change matter? I really hope it doesn't.

Reply:
no, just normalizing a bit with the ~matching invocation in test-cmd.sh, these went through a fair few iterations pinning down what was actually happening with these throughout the process tree.